### ISSUE TITLE
Bilobed right lung

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6230]
**Preferred term label:**
Bilobed right lung

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect of pulmonary lobation characterized by the presence of only two lobes of the right ling.

Comment: The normal anatomy of the right lung comprises three lobes: the superior, middle, and inferior lobes, and that of the left lung consists of two lobes: the superior and inferior lobes. 

@crapp, @mgriese, @katrinknoflach please revise definition as necessary

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002101

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/265380

PMID:19500772  1/14

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Added as HP:0033183 for upcoming release. Update definition as needed

**Comment by mgriese** [comment]
Edit just common wording:

Comment: The normal anatomy of the right lung comprises three lobes: the upper, middle, and lower lobes, and that of the left lung consists of two lobes: the upper and lower lobes.

**Comment by pnrobinson** [comment]
thanks. updated comment
